src/main/java/race/InputView.java

```diff
@@ -0,0 +1,75 @@
+package race;
+
+import java.util.Arrays;
+import java.util.InputMismatchException;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+public class InputView {
+    private final Scanner scanner;
+    private final String CAR_NAME_SPLIT_REGEX = "\\s*,\\s*";
+    private final int MIN_CAR_COUNT = 1;
+    private final int MAX_CAR_COUNT = 5;
+    private final int MIN_RACE_COUNT = 1;
+    private final int MAX_RACE_COUNT = 9;
+
+    public InputView(Scanner scanner) {
+        this.scanner = scanner;
+    }
+
+    public InputView() {
+        this(new Scanner(System.in));
+    }
+
+    public List<String> receiveCarNames() {
+        System.out.println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).");
+        String carNamesString = scanner.nextLine();
+        return Arrays.stream(carNamesString.split(","))
+                .map(String::trim)
+                .collect(Collectors.collectingAndThen(Collectors.toList(),
+                        list -> {
+                            validateCarNamesCount(list);
+                            return List.copyOf(list);
+                        }));
```

Comment:
```suggestion
                .collect(Collectors.collectingAndThen(Collectors.toUnmodifiableList(),
                        list -> {
                            validateCarNamesCount(list);
                            return list;
                        }));
```
`toUnmodifiableList`를 사용하면 리스트를 만들필요는 없을 것 같네요

Reply:
앗 그러네요! ㅎㅎ; 수정하겠습니다!